### DISCUSSION TITLE
Bare metal setup

### DISCUSSION THREAD

**Discussion by manuelkamp** [gotenberg/gotenberg#511] [Q&A]
How can I install gotenberg on bare metal? I do not want to run docker inside the LXC.

**Comment by gulien** [comment]
Hello, 

Not out of the box. But if you're into that, you may use the same instructions as in the Dockerfile here: https://github.com/gotenberg/gotenberg/blob/main/build/Dockerfile.

**Reply by manuelkamp** [reply]
Thanks, I do not have any clue about Docker and do not want to learn it - so I just need to install the things mentioned in the dockerfile?

**Reply by gulien** [reply]
Yep and create a binary from the Golang project.
